PLATER/services/util/graph_adapter.py

```diff
@@ -49,8 +49,8 @@ def ping(self):
         Pings the neo4j backend.
         :return:
         """
-        neo4j_db_labels_endpoint = "/db/data/labels"
-        ping_url = f"{self._scheme}://{self._host}:{self._port}{neo4j_db_labels_endpoint}"
+        neo4j_test_connection_endpoint = ""
+        ping_url = f"{self._scheme}://{self._host}:{self._port}/{neo4j_test_connection_endpoint}"
```

Comment:
does neo4j 3 have test_connection_endpoint in the http api?

Reply:
hi @kshefchek , Connecting to the root http endpoint in both 3.5 and 4.2 returns a json data on available routes. This let's us detect if webserver is up. To my knowledge there is no specific ping endpoint in 3.5+ or 4.x  versions of neo4j

Reply:
okay great, just checking for those of us in the dark ages